### PR TITLE
chore(image-deps): remove unused parts

### PR DESCRIPTION
**Pull request by eventualbuddha**
## Overview
Most of the code removed was used in ballot interpretation. Now that that's done in Rust, none of these were used anymore. This leaves the `image-utils` package with a fairly narrow set of responsibilities:
- cropping (only used in BMD interpretation and should be replaced/moved to Rust)
- loading images from files or `Buffer`s (especially but not exclusively for image fixtures)
- writing images to disk (especially but not exclusively for Jest image snapshots)

This PR also removes support for grayscale images within `image-utils`. While it's possible to create a `ImageData` from `canvas` with a `Uint8ClampedArray` that only has one byte per pixel (a grayscale image), none of the APIs for working with the `ImageData` actually support that, so we always have to convert to RGBA anyway. Now that most of the `image-utils` modules are gone, this is simpler to enforce. Going forward, any actual image manipulation should probably be done in Rust, or with a fairly narrow scope in JS and/or using the `canvas` APIs. Further, not all implementations of the canvas APIs allow creating such `ImageData`s (in particular [`@napi-rs/canvas`](https://github.com/Brooooooklyn/canvas)).

## Demo Video or Screenshot
n/a

## Testing Plan
Automated

### CODE REVIEW

**Comment by adghayes** [comment]
@eventualbuddha I'm going to do a pass tomorrow. I'm using some image utils for the fujitsu printer and want to make sure I'm not losing anything. I intend to rewrite what I need in rust but not immediately.

File: apps/scan/backend/src/scanners/custom/state_machine.ts [hunk 1]
```diff
@@ -332,13 +332,25 @@ async function scan({ client, workspace }: Context): Promise<SheetOf<string>> {
...
+      const luma = grayPixels[fromOffset];
+      rgbaPixels[toOffset] = luma;
+      rgbaPixels[toOffset + 1] = luma;
+      rgbaPixels[toOffset + 2] = luma;
+    }
+    const imageData = createImageData(
+      rgbaPixels,
+      image.imageWidth,
+      image.imageHeight
+    );
```
[the remark this answers is not in the document]
Comment (eventualbuddha) [reply apps/scan/backend/src/scanners/custom/state_machine.ts:353]: It probably should be separated out somewhere, yeah.

**Review by adghayes — APPROVED** [review]
LGTM, turns out I'm only using `pdfToImages`.

File: apps/scan/backend/src/scanners/custom/state_machine.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (eventualbuddha) [reply apps/scan/backend/src/scanners/custom/state_machine.ts:338]: Added `fromGrayScale` in [7d73787](https://github.com/votingworks/vxsuite/pull/4683/commits/7d73787f6b05d4f4ca4e4aa7e12b9f1ada1a76e5).

**Pull request merged**